scripts/custom_objects_spec.json

```diff
@@ -0,0 +1,554 @@
+{
+  "/apis/{group}/{version}/{kind}": {
+    "parameters": [
+      {
+        "uniqueItems": true,
+        "type": "string",
+        "description": "If 'true', then the output is pretty printed.",
+        "name": "pretty",
+        "in": "query"
+      },
+      {
+        "name": "group",
+        "in": "path",
+        "required": true,
+        "description": "The Custom object's group name",
```

Comment:
Some of the usages of "custom object" in this spec make more sense to me as "custom resource". We intentionally added the "D" in "CRD" to distinguish the custom resource from the thing that installs it (the custom resource definition).

The way I think of it is: the CRD installs a custom resource, which is the location in the API where you can store custom objects.